File: Upcoder/Other/Bài 3 Ôn Tập/On_LTCB3.cs
// Code này chưa chấm trên Upcoder
using System;
namespace Ôntap3
{
    public class Program
    {
        static void Main(string[]args)
        {
            string [] input = Console.ReadLine().Split();
            int a = int.Parse(input[0]);
            int b = int.Parse(input[1]);
            int c = int.Parse(input[2]);
            int max = a;
            if(max < b)
                max = b;
            if(max < c)
                max = c;
            int min = a;
            if(min > b)
                min = b;
            if(min > c)
                min = c;
            int medium = a;
            if ((b > min && b < max) || (b > max && b < min))
                medium = b;
            if ((c > min && c < max) || (c > max && c < min))
                medium = c;
            if (medium % 2 == 0)
                Console.Write("{0} la so chan", medium);
            else
                Console.Write("{0} la so le", medium);
        }
    }
}

File: Upcoder/Other/Bài 3 Ôn Tập/On_LTCB3.py
#code này chưa chấm trên upcoder
a, b, c = map(int, input().split())
Max = a
if(Max < b):
    Max = b
if(Max < c):
    Max = c
Min = a
if(Min > b):
    Min = b
if(Min > c):
    Min = c
medium = a
if ((b > Min and b < Max) or (b > Max and b < Min)):
    medium = b
if ((c > Min and c < Max) or (c > Max and c < Min)):
    medium = c
if (medium % 2 == 0):
    print(medium,"la so chan", end="")
else:
    print(medium,"la so le", end="")